en/advanced_config/pid_tuning_guide_multicopter.md

```diff
@@ -1,217 +1,185 @@
+> **Note** For safety reasons, the default gains are set to low values.  
+> You must increase the gains before you can expect good control responses. 
 
-First of all set all parameters to initial values:
+Here are some general points to follow when tuning:
+- All gains should be increased very slowly as large gains may cause dangerous oscillations! 
+  Typically increase gains by 20-30% per iteration, reducing to 5-10% for final fine tuning.
+- Land before changing a parameter. Slowly increase the throttle and check for oscillations.
+- Tune the vehicle around the hovering thrust point, and use the [thrust curve parameter or TPA](#thrust_curve) to account for thrust non-linearities or high-thrust oscillations.
 
-1. Set all `MC_XXX_P` to zero (ROLL, PITCH, YAW)
-2. Set all `MC_XXXRATE_P`, `MC_XXXRATE_I`, `MC_XXXRATE_D` to zero,
-   except [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) and [MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P)
-3. Set [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) and [MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P) to a small value, e.g. 0.02
-4. Set [MC_YAW_FF](../advanced_config/parameter_reference.md#MC_YAW_FF) to 0.5
+### Rate Controller
 
-> **Note** For safety reasons, the default gains are set to small value.  
-> You MUST increase the gains before you can expect any control responses. 
+The rate controller is the inner-most loop with three independent PID controllers to control the body rates:
 
-All gains should be increased very slowly, by 20%-30% per iteration, and
-even 10% for final fine tuning. Note, that too large gain (even only
-1.5-2 times more than optimal!) may cause very dangerous oscillations!
-
-
-### Step 2: Stabilize Roll and Pitch Rates
-
-#### P Gain Tuning
+- Roll rate control ([MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P), [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I), [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D))
+- Pitch rate control ([MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P), [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I), [MC_PITCHRATE_D](../advanced_config/parameter_reference.md#MC_PITCHRATE_D))
+- Yaw rate control ([MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P), [MC_YAWRATE_I](../advanced_config/parameter_reference.md#MC_YAWRATE_I), [MC_YAWRATE_D](../advanced_config/parameter_reference.md#MC_YAWRATE_D))
 
-Parameters: [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P), [MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P).
+A well-tuned rate controller is very important as it affects all flight modes.
+The difference between a badly and a well tuned rate controller is, for example, easily noticeable in [Position mode](../flight_modes/position_mc.md).

```

Comment:
This comment can lead to more confusion and it does not say much. Bad attitude and velocity control tuning will  be visible during position mode as well. It would be better to describe how bad rate control tuning is visible to the user during rate control, because bad rate control tuning should be visible in all modes.

Reply:
The intention is to highlight the importance of the rate controller for all higher modes. You are right that the other controllers are important as well, but I often see logs where the rate controller is very badly tuned, and I want to improve that.

Reply:
>  It would be better to describe how bad rate control tuning is visible to the user during rate control, because bad rate control tuning should be visible in all modes.

@bkueng I think that what @Stifael is saying is that it would be better to explain the *symptoms* for bad rate control, and I agree. If this is important then it should also be in a callout - ie warning, tip or note. 

So I would change to something like:

> **Note** A well-tuned rate controller is very important as it affects *all flight modes*. A badly tuned rate controller will be visible in [Position mode](../flight_modes/position_mc.md) (for example) as [Insert symptoms here].

Reply:
Yep that's better. Symtoms are: twitches, the vehicle will not stand perfectly still in the air.

Reply:
@hamishwillee yep thats what I meant.

Reply:
Great. Fixed inhttps://github.com/PX4/px4_user_guide/pull/272/commits/994d95c14305ce771d704cae8c650d19378e92e9